scripts/genny

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+# Runs the genny python wrapper in the venv which runs the binary.
```

Comment:
`set -eou pipefail` at the top and put `VIRTUAL_ENV_DISABLE_PROMPT=1` before the `source` to prevent errors about `$PS1` being undefined.

Reply:
Ah didn't know about the latter trick!